### PR TITLE
[change]: Update Appwrite Marketplace logo

### PR DESCRIPTION
**Pull request by hmorris3293**
## Description 📝
Update Appwrite to include new logo and colors
## Changes  🔄
upload appwrites new logo

## Target release date 🗓️
as soon as possible (no rush, but if we can fit it in next release that would make the partner happy :))

### CODE REVIEW

File: packages/manager/.env.example
Comment (hkhalil-akamai) [inline packages/manager/.env.example:1]: Whoops 😅

**Review by hkhalil-akamai — CHANGES_REQUESTED** [review]

**Comment by hmorris3293** [comment]
Before:
Screenshot 2024-07-30 at 3 51 26 PM
After:
Screenshot 2024-07-30 at 3 52 20 PM

**Comment by hkhalil-akamai** [comment]
> After: Screenshot 2024-07-30 at 3 52 20 PM

The way the wider aspect ratio causes the logo text to become illegible is not great -- I wonder if we could include just the circular icon and leave out the text?

**Comment by hmorris3293** [comment]
Screenshot 2024-07-31 at 10 33 46 AM
Updated screenshot with new logo

**Review by jdamore-linode — APPROVED** [review]
Thanks @hmorris3293!

**Review by hana-linode — APPROVED** [review]

**Review by hkhalil-akamai — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**